pkg/certs/cert-inspection/certgraphanalysis/collector.go

```diff
@@ -41,6 +48,7 @@ func isPlatformNamespace(nsName string) bool {
 }
 
 type configMapFilterFunc func(configMap *corev1.ConfigMap) bool
+type configMapRewriteFunc func(configMap *corev1.ConfigMap, nodes map[string]int) (string, string)
```

Comment:
This should rewrite the actual configmap so other readers all see the same information.  It should indicate that it mutated by some kind of annotation.